prow/cluster/BUILD

```diff
@@ -0,0 +1,28 @@
+load("@io_bazel_rules_k8s//k8s:object.bzl", "k8s_object")
+
+# Usage:
+#   bazel run //prow/cluster:foo.apply
+
+# TODO(fejta): consider k8s_deploy alias
+# https://github.com/bazelbuild/rules_k8s#aliasing-eg-k8s_deploy
```

Comment:
FWIW, I'm starting to bias away from `kind`.  Right now it's only used for `:foo.describe` and even that could probably key off of the information in the template.  Add in the explosion of aliases and it gets a bit unwieldy.

Reply:
I use describe all the time though?

Reply:
That's good info, so maybe we want a form of describe that extracts what it needs (and does the right thing for multiple objects) without needing kind here explicitly.

Reply:
I suppose I could get similar info from `:foo.get -o yaml` if that is supported? I mostly use describe to understand what is happening with the image after deploying